programs/bpf_loader/src/syscalls.rs

```diff
@@ -2933,6 +2933,10 @@ fn get_translated_accounts<'a, T, F>(
 where
     F: Fn(&T, &InvokeContext) -> Result<CallerAccount<'a>, EbpfError<BpfError>>,
 {
+    let transaction_context = &invoke_context.transaction_context;
+    let instruction_context = transaction_context
+        .get_current_instruction_context()
+        .map_err(SyscallError::InstructionError)?;
```

Comment:
Behavior change?

Reply:
Can't fail because:
- When performing CPI (from a BPF program) you will have at least one frame on the stack, so it can not be empty
- The exact same get_current_instruction_context() is performed earlier [here](https://github.com/solana-labs/solana/blob/7ee5094d831019608f25dae33d60185e25cb5ff1/programs/bpf_loader/src/syscalls.rs#L3124) as well